mipengine/node/tasks/remote_tables.py

```diff
@@ -6,6 +6,10 @@
 from mipengine.node.monetdb_interface import remote_tables
 
 
+# TODO: the database name has to come from somewhere outside this module. Not from the controller, controller should not know that
+DB_NAME = "db"
```

Comment:
We could add the `DB_NAME` on the config file instead of having it here hard coded.